### PR TITLE
samples: wifi: thread_coex: Readme update

### PR DESCRIPTION
**Pull request by muraliThokala**
Minor edits to suit the latest documentation style.

### CODE REVIEW

**Review by richabp — CHANGES_REQUESTED** [review]

File: samples/wifi/thread_coex/README.rst [hunk 1]
```diff
@@ -120,11 +92,11 @@ To enable different test modes, set up the following configuration parameters in
...
 
 .. note::
-   ``menuconfig`` can also be used to configure ``Wi-Fi credentials``
+   ``menuconfig`` can also be used to configure ``Wi-Fi credentials``.
 
-* Wi-Fi throughput test: Set the :kconfig:option:`CONFIG_NET_CONFIG_PEER_IPV4_ADDR` Kconfig option appropriately as per the Wi-Fi interface IP address of the test PC on which **iperf** is run.
+     See :ref:`zephyr:menuconfig` in the Zephyr documentation for instructions on how to run ``menuconfig``.
 
-See :ref:`zephyr:menuconfig` in the Zephyr documentation for instructions on how to run ``menuconfig``.
+* Wi-Fi throughput test: Set the :kconfig:option:`CONFIG_NET_CONFIG_PEER_IPV4_ADDR` Kconfig option appropriately as per the Wi-Fi interface IP address of the test PC on which **iperf** is run.
```
[the remark this answers is not in the document]
Comment (muraliThokala) [reply samples/wifi/thread_coex/README.rst:99]: 1. Wi-Fi connection is done before running the Wi-Fi throughput test. So, these two are in this order in the doc. 
2. Configuring the credentials is part of the Wi-Fi connection and so mentioned immediately after the Wi-Fi connection, Alternative way of configuring the credentials (using menuconfig)  is mentioned as part of this.
So, connection, configuring credentials and running throughput are in the right sequence.

[the remark this answers is not in the document]
Comment (richabp) [reply samples/wifi/thread_coex/README.rst:99]: Ok, thanks!

**Review by richabp — APPROVED** [review]

**Review by srkanordic — APPROVED** [review]

File: samples/wifi/thread_coex/README.rst [hunk 2]
```diff
@@ -292,8 +265,14 @@ Testing
 	|               |             | on client** is seen on the DUT nRF7002 DK's UART console window.        |
 	+---------------+-------------+-------------------------------------------------------------------------+
 
-#. Observe that the Wi-Fi throughput result appears on the test PC terminal on which **iperf** server is run.
-   The Thread throughput result appears on the minicom terminal connected to the nRF7002 DK, on which Thread is run in the client role.
+When the sample runs Wi-Fi UDP throughput in client mode, a peer device (test PC) runs UDP throughput in server mode using the following command:
+
+	.. code-block:: console
+
+	   $ iperf -s -i 1 -u
```
Comment (peknis) [inline samples/wifi/thread_coex/README.rst:272]:
```suggestion
.. code-block:: console

   $ iperf -s -i 1 -u
```

File: samples/wifi/thread_coex/README.rst [hunk 3]
```diff
@@ -292,8 +265,14 @@ Testing
 	|               |             | on client** is seen on the DUT nRF7002 DK's UART console window.        |
 	+---------------+-------------+-------------------------------------------------------------------------+
 
-#. Observe that the Wi-Fi throughput result appears on the test PC terminal on which **iperf** server is run.
-   The Thread throughput result appears on the minicom terminal connected to the nRF7002 DK, on which Thread is run in the client role.
+When the sample runs Wi-Fi UDP throughput in client mode, a peer device (test PC) runs UDP throughput in server mode using the following command:
```
Comment (peknis) [inline samples/wifi/thread_coex/README.rst:268]: Does the user need to run that command or does it just happen?
If the user needs to run it, then rather:
"... runs UDP throughput in server mode.
Use the following command:"

File: samples/wifi/thread_coex/README.rst [hunk 4]
```diff
@@ -214,35 +186,36 @@ Testing
...
-	When the sample runs Wi-Fi UDP throughput in client mode, a peer device (test PC) runs UDP throughput
-	in server mode using the following command:
+	  $ west flash --dev-id <device-id> --hex-file build/merged.hex
 
-	.. code-block:: console
+Test procedure
+==============
 
-	   $ iperf -s -i 1 -u
+Tables given below provide the procedure to run Wi-Fi only, Bluetooth LE-only, and combined throughput for different Thread roles.
```
Comment (peknis) [inline samples/wifi/thread_coex/README.rst:218]:
```suggestion
The following tables provide the procedure to run Wi-Fi only, Bluetooth LE-only, and combined throughput for different Thread roles.
```

File: samples/wifi/thread_coex/README.rst [hunk 5]
```diff
@@ -214,35 +186,36 @@ Testing
...
 	   $ minicom -D /dev/ttyACM3 -b 115200
 
-#. Program the nRF7002 DK by completing the following steps:
+Programming DKs
+===============
 
-	1. |open_terminal_window_with_environment|
-	#. Navigate to the :file:`<ncs code>/nrf/samples/wifi/thread_coex/` folder.
-	#. Run the following command:
+Program the nRF7002 DK by completing the following steps:
```
Comment (peknis) [inline samples/wifi/thread_coex/README.rst:205]:
```suggestion
Complete the following steps to program the nRF7002 DK:
```

File: samples/wifi/thread_coex/README.rst [hunk 6]
```diff
@@ -214,35 +186,36 @@ Testing
       1050759502         /dev/ttyACM3    VCOM1
       $
 
-	In this example, ``1050779496`` is the device ID of the first nRF7002 DK and ``1050759502`` is the device ID of the second nRF7002 DK.
+	  In this example, ``1050779496`` is the device ID of the first nRF7002 DK and ``1050759502`` is the device ID of the second nRF7002 DK.
```
Comment (peknis) [inline samples/wifi/thread_coex/README.rst:189]:
```suggestion
	  In this example, ``1050779496`` is the device ID of the first nRF7002 DK and ``1050759502`` is the device ID of the other one.
```

**Comment by peknis** [comment]
To improve the readability of the table under the test setup image, I'd have an empty line between the two cases. (line 58).

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by peknis — APPROVED** [review]

File: samples/wifi/thread_coex/README.rst [hunk 7]
```diff
@@ -292,8 +265,14 @@ Testing
 	|               |             | on client** is seen on the DUT nRF7002 DK's UART console window.        |
 	+---------------+-------------+-------------------------------------------------------------------------+
 
-#. Observe that the Wi-Fi throughput result appears on the test PC terminal on which **iperf** server is run.
-   The Thread throughput result appears on the minicom terminal connected to the nRF7002 DK, on which Thread is run in the client role.
+When the sample is executed, the Wi-Fi UDP throughput operates with the Device Under Test (DUT) in client mode. To run the UDP throughput in server mode on the peer device (test PC), use the following command.
```
Comment (richabp) [inline samples/wifi/thread_coex/README.rst:268]:
```suggestion
When the sample is executed, the Wi-Fi UDP throughput operates with the Device Under Test (DUT) in client mode.
To run the UDP throughput in server mode on the peer device (test PC), use the following command.
```

**Review by richabp — CHANGES_REQUESTED** [review]

**Review by richabp — APPROVED** [review]

**Pull request merged**